packages/opal-common/opal_common/utils.py

```diff
@@ -136,6 +138,7 @@ def getSignatureKey(key, dateStamp, regionName, serviceName):
 
     return {
         "x-amz-date": amzdate,
+        "x-amz-content-sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
```

Comment:
hi @kbalthaser

what is this value? is this a secret? if so - we do not allow secrets in the codebase.
otherwise it should sit in a config var (if value can be changed by users) or in a constant.

Reply:
It is the checksum of an empty string, as required by the spec when doing an api operation with no payload. 

I can move it to a constant, as it never changes in this context.